docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md

```diff
@@ -64,23 +64,6 @@ Service Fabric is a good choice if you're creating a new app or rewriting an exi
 
 If you have an existing application that would require substantial modifications to run in App Service or Service Fabric, you could choose Virtual Machines in order to simplify migrating to the cloud. However, correctly configuring, securing, and maintaining VMs requires much more time and IT expertise compared to Azure App Service and Service Fabric. If you're considering Azure Virtual Machines, make sure you take into account the ongoing maintenance effort required to patch, update, and manage your VM environment. Azure Virtual Machines is infrastructure as a service (IaaS), while App Service and Service Fabric are PaaS.
 
-#### Feature Comparison
```

Comment:
I guess this change was included by mistake? Since this doc is part of Modern web apps eBook and there is already a PR there, it's best to discard these changes here.

Reply:
Yeah, probably did it while on the wrong branch. I'll confirm the change is in the other PR.

Reply:
@nishanil fixed so each PR only affects its respective eBook.